apps/examples/st_things/light/sample_light_LED.c

```diff
@@ -0,0 +1,343 @@
+/****************************************************************************
+*
+* Copyright 2019 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+****************************************************************************/
+
+#include <tinyara/config.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include <ctype.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/ioctl.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+
+#include <signal.h>
+
+#include <tinyara/gpio.h>
+#include <tinyara/fs/fs.h>
+
+#include "sample_light_LED.h"
+
+const char * gpio_path_format = "/dev/gpio%d";
+
+
+int key_get_status(int key, int* status)
+{
+	int fd_key = -1;
+	int ret = ERROR;
+	char path[16] = { 0 };
+
+	if (key == KEY_RESET_IO_NUM) {
+		sprintf(path, gpio_path_format, KEY_RESET_IO_NUM);
+	}
```

Comment:
Please make it like: } else if {

Reply:
I fixed all coding rule problems, thanks for your review!